.changes/unreleased/tinymce-TINY-11997-2025-04-14.yaml

```
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Fixed
+body: Tabbing inside a `figcaption` didn't remove the fakeCaret.
```

Implementation details don't really belong in changelogs 🤔 
```suggestion
body: Tabbing inside a `figcaption` element no longer shows two text insertion carets.
```

---

done